cpg_workflows/stages/joint_genotyping.py

```diff
@@ -59,19 +50,19 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
             raise WorkflowError(f'Joint genotyping: could not find {len(not_found_gvcfs)} GVCFs, exiting')
 
         jobs = []
-        vcf_path = self.expected_outputs(cohort)['vcf']
-        siteonly_vcf_path = self.expected_outputs(cohort)['siteonly']
-        scatter_count = joint_calling_scatter_count(len(cohort.get_sequencing_groups()))
+
+        # create expected outputs once
+        outputs = self.expected_outputs(multicohort)
+        scatter_count = joint_calling_scatter_count(len(gvcf_by_sgid))
         out_siteonly_vcf_part_paths = [
-            to_path(self.expected_outputs(cohort)['siteonly_part_pattern'].format(idx=idx))
-            for idx in range(scatter_count)
+            to_path(outputs['siteonly_part_pattern'].format(idx=idx)) for idx in range(scatter_count)
         ]
 
         jc_jobs = joint_genotyping.make_joint_genotyping_jobs(
             b=get_batch(),
-            out_vcf_path=vcf_path,
-            out_siteonly_vcf_path=siteonly_vcf_path,
-            tmp_bucket=to_path(self.expected_outputs(cohort)['tmp_prefix']),
+            out_vcf_path=outputs['vcf'],
+            out_siteonly_vcf_path=outputs['siteonly'],
+            tmp_bucket=self.prefix / 'tmp',
```

Comment:
Should we add a comment here why it is not going to the temp bucket?

Reply:
Ugh, I've made this into a real TMP path - we should review the stuff that's already being retained (e.g. [here](https://console.cloud.google.com/storage/browser/cpg-seqr-main/seqr_loader/6c662c33c550a9c69d55c909c1388cc7e4930b_3755/JointGenotyping/tmp)) - looks like we're keeping multiple terabytes of additional data per joint call that should probably have been in tmp